tests/src/tracing/common/NetperfFile.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace Tracing.Tests.Common
+{
+    public class NetperfFile : IDisposable
```

Comment:
`NetperfFile` => `NetPerfFile` (File name too.)

Reply:
I never really considered the casing of the netperf extension (I always use all lowercase) - @adamsitnik I'll defer to you on your preference here.